test/evergreen.yml

```diff
@@ -923,6 +923,19 @@ tasks:
 
   # End of Python unit test tasks
 
+  - name: s-all
```

Comment:
Depends on if WT-5176 is merged before this one, we need to tag this task as `pull-request`.

Reply:
We can fist merge WT-5176 and rebase.